### PR TITLE
fix(curriculum): missing `:root` test to skyline css challenge

### LINKED ISSUE #46385: Missing test in step 117 of city skyline project
### Describe the Issue

missing test in step 117 of city skyline project
the code in the file is like this
```
> # --hints--
> 
> You should add a `:root` selector to the media query.
> 
> ```js
> ```
```

### Affected Page

https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e993d.md

### Your code

```

```

### Expected behavior

The missing test should be added to the file

### Screenshots

_No response_

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Pegasoos**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #46385

### CODE REVIEW

**Review by Sboonny — COMMENTED** [review]
Unfortunately, if I type nothing, I don't get the first hint `You should add a :root selector to the media query`

Screenshot_2022-06-09-21-40-53-99_e4424258c8b8649f6e67d283a50a2cbc

---

If you are facing issue, we are happy to help in contribution section in the https://forum.freecodecamp.org , or feel free to drop a message in https://chat.freecodecamp.org/channel/contributors

File: curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e993d.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ Add a `:root` selector to the top of your media query. Then redefine all four of
 You should add a `:root` selector to the media query.
 
 ```js
+assert.equal(new __helpers.CSSHelp(document).getRuleListsWithinMedia("(max-width: 1000px)")?.find(x=>x.selectorText === ":root"));
```
Comment (gikf) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e993d.md:17]:
```suggestion
assert.notDeepEqual(new __helpers.CSSHelp(document).getRuleListsWithinMedia("(max-width: 1000px)")?.find(x=>x.selectorText === ":root"), undefined);
```
One way of fixing it

**Review by Sboonny — APPROVED** [review]
Hi @Pergamon256 . Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Review by gikf — APPROVED** [review]
Looks good :+1:

**Pull request merged**